CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 # Default owners for all files
 
-- @gidjin @haworku @ahobson @brandonlenz @sirenaborracha @kylehilltruss @rogeruiz @jcbcapps
+* @gidjin @haworku @ahobson @brandonlenz @sirenaborracha @kylehilltruss @rogeruiz @jcbcapps
```

Comment:
What prompted this change? Was it our pre-commit configuration around Markdown linting?

Reply:
I'm not sure, I think someone's editor _helpfully_ converted markdown bullet points from `*` to `-` but turns out `CODEOWNERS` isn't a markdown file and `*` is referring to all files and `-` means nothing. This happened on my current project too, I suspect VSCode or something similar. But I have no actual proof of such things but this is the fix for why our PRs are not requiring owners at the moment.